src/en/wizden-staff/maintainer/review-procedure.md

```diff
@@ -1,54 +1,184 @@
+
+Note that PRs undergoing maintainer discussion or awaiting review are not considered stale.
+
+### Drafts
+
+PRs should not be marked as drafts unless there is a meaningful reason behind them.
+Simply opening an incomplete PR as a draft is not permitted and will result in immediate closure.
+
+Drafts are only permitted if a portion of the PR requires review and/or approval in order for other segments of the same PR to be completed.
+Note that this does not apply if the PR is able to be easily atomized.
+
+## Discussion Policy
+
+Maintainer discussions are an optional way to gain insight into general staff sentiment.
+They can be used both when there are disagreements regarding a PR and when a maintainer is not confident in the design of a PR.
+
+PRs actively undergoing a maintainer discussion should not be merged.
+
+To begin the discussion process, add the `S: Undergoing Maintainer Discussion` tag.
+The maintainer should then create a post in the `#maint-reviews` discord channel with the PR's name, number, link, and appropriate tags.
+
+Discussion can then proceed with maintainers and admins giving opinions and coming to a conclusion or compromise.
+Once a conclusion is reached or regular discussion ceases, one of the following must occur:
+- If the discussion was created due a disagreement in the PR decision,
+
+
+- - If a definitive decision was reached (Approve/Close), then it should be acted upon by a maintainer.
+
+
+- - If a compromise within the scope of the PR is reached, then the PR should be approved once the compromise is implemented.
+Inability/refusal to implement the compromise should result in the closure of the PR.
+
+
+- - If no definitive decision was reached, then the PR should be closed.
+
+  
+- If the discussion was _not_ created due to a disagreement in the PR decision,
+
+
+- - If a definitive decision or reasonable compromise was reached, then it should be acted upon and/or implemented.
+
+
+- - If no definitive decision was reached, the maintainer who created the discussion may choose to resolve the PR as they see fit. 
+
+
+A positive conclusion should be indicated on the PR with the `S: Conceptual Approval` tag.
+This replaces the need for 2 design approvals (though the code still needs to be separately reviewed) and allows the PR to be merged.
+
+If the discussion has a negative conclusion, the closure message for the PR should include a brief summary of the discussion.
+This should have information about the elements that need to be addressed if a subsequent PR were to be made.
+
+## Review Guidelines
+The following are general guidelines to follow when reviewing PRs. 
+These are especially important to follow for first-time contributors.
+
+- Use **constructive** language and avoid being overly negative.
+
+
+- Try not to be _overly_ vulgar or swear in a derogatory way.
+
+
+- Avoid comments that personally criticize the author.
+It's okay to critique code and decisions, but you shouldn't insult their character.
```

Comment:
There might not need to be a change here but general code critiquing etiquette is definitely something that needs to be addressed at some point. There have been numerous times where wording was used I find generally unacceptable but got dismissed as not being a problem.

Reply:
this is really meant to just  be general tips. If we want anything more specific i think it would be best to put it in its own spot.